Android-Contributions/MemeShare/app/src/main/res/layout/activity_main.xml

```diff
@@ -33,32 +33,34 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-    <Button
+    <androidx.appcompat.widget.AppCompatButton
         android:id="@+id/btnNext"
-        android:layout_width="0dp"
+        android:layout_width="100dp"
         android:layout_height="wrap_content"
         android:layout_margin="10dp"
         android:height="100dp"
         android:background="@drawable/circular_button"
-        android:backgroundTint="@color/teal_200"
+        android:backgroundTint="@null"
         android:padding="15dp"
         android:text="NEXT"
-        android:textSize="22sp"
```

Comment:
It is suggested to do not hard code any dimension values, 

Just create one dimen.xml file in the values folder and then use that

Reply:
ok